docs/src/client/template.html

```diff
@@ -2,6 +2,15 @@
 <html lang="en">
 	<head>
 		<meta charset="utf-8">
+
+		<!-- OneTrust Cookies Consent Notice start for bestjs.dev -->
+		<script src="https://a.sfdcstatic.com/enterprise/bestjs/uat/6140/oneTrust/scripttemplates/otSDKStub.js"  type="text/javascript" charset="UTF-8" data-domain-script="869194d9-5b37-4d66-b7c1-dc6d705efb5f" ></script>
```

Comment:
Move this to the end of the page or mark them `async`. Both scripts are currently parser blocking, which is bad of performance.

**Edit:** I didn't realized this, but we have a similar issue with lwc.dev